modules/ztp-acm-preparing-to-install-disconnected-acm.adoc

```diff
@@ -11,3 +11,14 @@ Before you can provision distributed units (DU) at scale, you must install {rh-r
 
 You also use a disconnected mirror host to serve the {op-system} ISO and RootFS disk images that provision the DU bare-metal host operating system.
 
+Before you install a cluster on infrastructure that you provision in a restricted network, you must mirror the required container images into that environment. You can also use this procedure in unrestricted networks to ensure your clusters only use container images that have satisfied your organizational controls on external content.
+
+[IMPORTANT]
+====
+You must have access to the internet to obtain the necessary container images.
+In this procedure, you place the mirror registry on a mirror host
+that has access to both your network and the internet. If you do not have access
+to a mirror host, use the disconnected procedure to copy images to a device you
+can move across network boundaries with.
```

Comment:
Could restructure to avoid the ending "with." Or not. NBD. :man_shrugging:

Reply:
done.